### ISSUE TITLE
Configure Nabla parameters

### ISSUE THREAD

**Issue by prayagd** [pendulum-chain/pendulum#457]
## Context
This ticket is to define the Nabla smart contract parameters for the mainnet

## Requirements
Following is the list of parameters and values to be defined for the smart contract

**Comment by prayagd** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Pendulum-6408a0365cda59c57d39bee5/issues/pendulum-chain/pendulum/457?planning-poker) with Zenhub @b-yap @bogdanS98 @ebma @gianfra-t @TorstenStueber

**Comment by prayagd** [comment]
@pendulum-chain/devs Please let me know if any question or anything not clear, Product team is clarifying the slippage comments [here](https://www.notion.so/satoshipay/Forex-AMM-launch-on-Pendulum-c019fca1850b4801968b011064e91ad0?pvs=4#baa902a2e9134526890b3ef68253630f) with Torsten, will update if there are any changes there or additional details can be provided. In light of the aim launch date for Nabla next week, wanted this ticked to be estimated and ready.

**Comment by ebma** [comment]
Why did you choose such a high timelock of 14 days? 
Also, we need to define 'backstop fee', 'LP fee' and 'protocol fee'. I'm not sure how the mentioned 'Swap fee' relates to these. I guess it's the 'protocol fee' but please confirm this again @pendulum-chain/product. If I understand correctly, the 'insurance fee' is synonym to the 'backstop fee'. The value for the LP fee is also missing from the description.

**Comment by prayagd** [comment]
> Why did you choose such a high timelock of 14 days?

This is from the assumption and understanding that most staking locks are also around 14 days as security measure so just a learning from there, we can reduce it in the iterative version if we receive feedback that its very long. 

> Also, we need to define 'backstop fee', 'LP fee' and 'protocol fee'.

As per my understanding LP fee is swap fee, protocol fee is treasury fee and you are right backstop fee is not defined is this assumption right?
Regarding backstop fee we didnt define because it was not provided in the parameter list earlier by tech but thanks for highlighting will look into it and get back.

**Comment by vadaynujra** [comment]
@ebma can you share some more context around why you think 14 days is a lot? Nabla's current deployment has this value at 60 days, we just aligned to our unstaking period and acknowledged that having this value very low will lead to lower sticky liquidity of the BSP.
@prayagd isn't the insurance fee = backstop fee, which is defined?

**Comment by ebma** [comment]
> can you share some more context around why you think 14 days is a lot? Nabla's current deployment has this value at 60 days, we just aligned to our unstaking period and acknowledged that having this value very low will lead to lower sticky liquidity of the BSP.

To which deployment are you referring to? I only saw the values we used in [previous deployments](https://www.notion.so/satoshipay/24-05-16-Nabla-Pendulum-Deployment-c7cefd58f44c428a92ceaea30c603286?pvs=4#7e1fb51b348d4f2b9375aea866ee7f4d) and they are a lot lower with just 1 day. Which doesn't mean we need to keep sticking to low values here. Maybe you have more insights also talking to Aaron, so I was curious why you decided to go with that value.

Can you please add a link to the relevant Notion page to the issue ticket?

Regarding the terminology of the fees, I would appreciate it if we could refer to them as the names used internally. As you can see, they are defined as 'LP fee', 'backstop fee', and 'protocol fee' [here](https://github.com/NablaFinance/contracts/blob/c35009ea1a8b47d2079a83b21d94fc809981265a/contracts/src/core/SwapPool.sol#L39-L41) and if I were to configure something about it as part of this ticket, I wouldn't know just from reading the description of the ticket. 
If 'swap fee' means 'LP fee', 'treasury fee' means 'protocol fee', and 'insurance fee' means 'backstop fee' then let's mention both in the description.

**Comment by prayagd** [comment]
> Regarding the terminology of the fees, I would appreciate it if we could refer to them as the names used internally.

Can do that, we were referred to [these](https://github.com/pendulum-chain/tasks/issues/246#issuecomment-1954012728) names and now that i read between the lines there the swap fees comprises of LP fee, protocol fee and backstop fee

**Comment by prayagd** [comment]
Split added in the notes section for the swap fee i.e for LP fee, BSP fee and protocol fee

**Comment by ebma** [comment]
Are we using the same swap fees for each asset then? And the table below '5. Swap fees (see notes section for the split)' is irrelevant? Or how do we need to interpret this?

**Comment by prayagd** [comment]
The split is 5:25:70 Protocl fee:LP fee: BSP fee, and we rounded up the numbers but i see your point i have provided only the stable split will also add the crypto split there

**Comment by ebma** [comment]
Great, thanks 👍

**Comment by prayagd** [comment]
Ticket updated for two data points for the slippage parameters as discussed in the tech sync

**Comment by prayagd** [comment]
Putting this on-hold as there are changes made to the swap pool caps

**Comment by vadaynujra** [comment]
@TorstenStueber think this will be done today? I need to inform Finance and Marketing about the new launch date.

**Comment by vadaynujra** [comment]
@prayagd @TorstenStueber since we lowered the initial liquidity we will be adding from outside by about 1/4, while keeping the pool caps the same, would it make sense to increase the max CR to a higher value e.g. 300% or 400%?

**Comment by prayagd** [comment]
@vadaynujra I dont agree to the suggestion as discussed on a huddle that increasing the CR with the current max caps in place would also decrease the backstop pool coverage of the pools.

**Comment by vadaynujra** [comment]
Okay so no changes to the max CR then @TorstenStueber please go ahead and can you confirm if you will do this today as I need to share updated timelines with Finance and Marketing?

**Comment by vadaynujra** [comment]
@prayagd @TorstenStueber turns out there is a max cap for BSP. Let's go with $20k USDC?

**Comment by TorstenStueber** [comment]
All fine with me.
